.gitignore

```diff
@@ -1,4 +1,6 @@
 build
```

Comment:
Should this `build` be removed?

Reply:
Hmm, now I worry slightly about someone with an existing repo, discovering a new `build` directory marked unknown.  But maybe that's good, encouraging people to delete it.  (`bower` might even have an issue with directory `build` existing, now that it's removed from other lists?)

Reply:
@edemaine I originally thought this ignores `website/build/` generated by #1484 too, but its PR adds it as separate entry. I'll remove it.